minecraft_servers/rangemc/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "server_name": "rangemc",
+  "nice_name": "RangeMC",
+  "direct_ip": "rangemc.net",
+  "supported_languages": [
+     "en",
+     "de"
+  ],
+  "social": {
+    "web": "https://rangemc.net",
+    "web_shop": "https://rangemc.net/store",
+    "web_support": "https://rangemc.net/support",
```

Comment:
Cannot be hidden behind a login mask; provide a classic faq or contact form or remove